python/src/Wilks_doc.i.in

```diff
@@ -1,66 +1,101 @@
 
 Parameters
 ----------
-randomVector : :class:`~openturns.RandomVector` of dimension 1
-    Output variable of interest.
+X : :class:`~openturns.RandomVector`,
+    A random vector of dimension 1.
 
 Notes
 -----
-This class is a static class which enables the evaluation of the Wilks number:
-the minimal sample size :math:`N_{\alpha, \beta, i}` to perform in order to
-guarantee that the empirical quantile :math:`\alpha`, noted
-:math:`\tilde{q}_{\alpha} N_{\alpha, \beta, i}` evaluated with the
-:math:`(n - i)^{th}` maximum of the sample, noted :math:`X_{n - i}` be greater
-than the theoretical quantile :math:`q_{\alpha}` with a probability at least
-:math:`\beta`:
+This static class allows one to estimate an upper bound of the quantile of order :math:`\alpha`  of the random variable :math:`X`
+with a confidence greater than :math:`\beta`, using a given order statistics.
+
+We denote by :math:`x_{\alpha}` the unknown quantile of order :math:`\alpha` of the random 1variable :math:`X` of dimension 1.
+
+We denote by :math:`(X_1, \dots, X_\sampleSize)` some independent and identically distributed variables according to :math:`X`.
+
+Let :math:`X_{(k)}` be the :math:`k` -th order statistics of :math:`(X_1, \dots, X_\sampleSize)` which means that
+:math:`X_{(k)}` is the :math:`k` -th maximum of :math:`(X_1, \dots, X_\sampleSize)` for :math:`1 \leq k \leq \sampleSize`. For
+example, :math:`X_{(1)} = \min (X_1, \dots, X_\sampleSize)` is the minimum
+and :math:`X_{(\sampleSize)} = \max (X_1, \dots, X_\sampleSize)` is the maximum. We have:
 
 .. math::
 
-    \Pset (\tilde{q}_{\alpha} N_{\alpha, \beta, i} > q_{\alpha}) > \beta
+    X_{(1)} \leq X_{(2)} \leq \dots \leq X_{(\sampleSize)} 
+
+Given :math:`\alpha`, :math:`\beta` and :math:`i`, the class estimates the minimal size :math:`\sampleSize` such that:
+
+.. math::
 
-where :math:`\tilde{q}_{\alpha} N_{\alpha, \beta, i} = X_{n-i}`."
+    \Prob{X_{(\sampleSize-i)} \geq x_{\alpha}} \geq \beta
+
+Once the minimal size :math:`\sampleSize`  has been estimated, a sample of size :math:`\sampleSize` can be generated from
+:math:`X` and an upper bound of :math:`x_{\alpha}` is estimated by the value of the :math:`X_{(\sampleSize-i)}` on the sample."
 
 // ---------------------------------------------------------------------
 
 %feature("docstring") OT::Wilks::ComputeSampleSize
-"Evaluate the size of the sample.
+"Evaluate the minimum size of the sample.
 
 Parameters
 ----------
 alpha : positive float :math:`< 1`
-    The order of the quantile we want to evaluate.
-beta : positive float :math:`< 1`
-    Confidence on the evaluation of the empirical quantile.
+    Order :math:`\alpha` of the quantile.
+beta : positive float :math:`< 1`,
+    Confidence on the upper bound.
 i : int
-    Rank of the maximum which will evaluate the empirical quantile. Default
-    :math:`i = 0` (maximum of the sample)
+    Index such that :math:`X_{(\sampleSize -i)}` is an upper bound of :math:`x_{\alpha}`
```

Comment:
This is not an _upper_, but a _lower_ bound: see the equation at line 66.
``suggestion
    Index such that :math:`X_{(\sampleSize -i)}` is a lower bound of :math:`x_{\alpha}`
```